src/components/views/elements/TooltipTarget.tsx

```diff
@@ -44,6 +44,19 @@ const TooltipTarget: React.FC<IProps> = ({
     const show = () => setIsVisible(true);
```

Comment:
TODO: Need to fix up `TooltipTarget` tests, https://buildkite.com/matrix-dot-org/matrix-react-sdk/builds/21267#c841717a-6dcb-4fed-84b2-93fd21390aae